test/odbc/psqlodbc/test_pg_datatype_nchar.cpp

```diff
@@ -0,0 +1,1019 @@
+#include <gtest/gtest.h>
```

Comment:
Please remove this whole file? It doesn't seem like it's related to sys.ntext in this PR